src/System.Composition/perftests/project.json

```diff
@@ -11,7 +11,7 @@
     "System.Runtime.Extensions": "4.0.10",
     "System.Threading": "4.0.10",
     "System.Threading.Tasks.Parallel": "4.0.0",
-    "System.Threading.Thread": "4.0.0-rc3-23910"
+    "System.Threading.Thread": "4.0.0-rc3-23918"
   },
   "frameworks": {
     "dnxcore50": {}
```

Comment:
yeah, test projects still have build tools infra that depends on dnxcore50.  This change was large enough without requiring a buildtools update.